bokeh/models/widgets/inputs.py

```diff
@@ -152,6 +152,10 @@ class Slider(InputWidget):
     A callback to run in the browser whenever the current Slider value changes.
     """)
 
+    delay_callback = Bool(default = False, help="""
```

Comment:
> Not sure why the spaces cause issues though. I think they help for readability.

This is the style we currently use and we would prefer the code base to stay consistent. If we decide to write spaces around `=`, we will change all instances and then ask submitters to stay consistent with the new style. In general you should follow the style and other rules of the surrounding code. We have fairly lax attitude here, but there are projects where every single mismatch will be required to be fixed.